Demo/Recycling/ListViews/UserAds/UserAdsListViewDemoViewHelpers.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+public struct UserAdHeaderCell: UserAdsListViewHeaderModel, Hashable {
+    public let title: String
+    public let buttonTitle: String
+
+    public var accessibilityLabel: String {
+        var message = title
+        message += ". " + buttonTitle
+        return message
+    }
+
+    public init(title: String = "", buttonTitle: String = "") {
+        self.title = title
+        self.buttonTitle = buttonTitle
+    }
+}
+
+public struct UserAdCell: UserAdsListViewModel {
+    public let imagePath: String?
+    public let imageSize: CGSize
+    public let title: String
+    public let price: String
+    public let detail: String
+    public let status: String
+
+    public var accessibilityLabel: String {
+        var message = title
+        message += ". " + price
+        message += ". " + detail
+        message += ". " + status
+        return message
+    }
+
+    public init(imagePath: String? = nil, imageSize: CGSize = CGSize(width: 0, height: 0),
+                title: String = "", price: String = "", detail: String = "", status: String = "") {
+        self.imagePath = imagePath
+        self.imageSize = imageSize
+        self.title = title
+        self.price = price
+        self.detail = detail
+        self.status = status
+    }
+}
+
+public struct UserAdsFactory {
+    private struct ImageSource {
+        let path: String
+        let size: CGSize
+    }
+
+    public static func createAds() -> [Int: (header: UserAdHeaderCell, ads: [UserAdCell])] {
+        let newAd = createNewAd()
+        let activeAds = createActiveAds()
+        let inactiveAds = createInactiveAds()
+        let seeAllAds = createSeeAllAds()
+        return [0: newAd,
```

Comment:
Isn't that enough to have an array and then use indices instead of dictionary keys?